cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTask.java

```diff
@@ -197,6 +203,20 @@ public void completed(long now) {
     _endTime = now;
   }
 
+  /**
+   * Check whether the execution time of the task is too long.
+   */
+  public boolean shouldReportExecutionTooSlow(long now) {
```

Comment:
Looks like after reporting this once, we always want to mute. Does it make sense to move (1) the logic regarding the reporting functionality, and (2) muting the task inside this method? -- i.e. a change from `shouldReportExecutionTooSlow` to `reportExecutionTooSlow`?